eng/Version.Details.xml

```diff
@@ -93,43 +93,43 @@
       <Sha>20cca61a546fe378948f0550a0026ec6077c1600</Sha>
       <SourceBuild RepoName="fsharp" ManagedOnly="true" />
     </Dependency>
-    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="4.12.0-3.24458.2">
+    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="4.12.0-3.24459.1">
```

Comment:
Shouldn't these version changes come through Arcade code flow?

Reply:
It shouldn't hurt to include dependency updates in your PR if your code depends on an external repo's changes that have not yet flowed to your current repo. It's ok as long as this file was modified using:

```
darc update-dependencies --channel <CHANNEL>
```

Ideally, a separate deps flow show come into the sdk repo with these same changes, and this PR can be rebased on top of it.

Reply:
Same as https://github.com/dotnet/sdk/pull/42861#discussion_r1758509761, this will go away before merging - we will need to merge the roslyn part first, then let it flow into SDK, then we will be able to remove these version/nuget changes.